lib/python/qmk/tests/test_qmk_path.py

```diff
@@ -1,13 +1,14 @@
 import os
+from pathlib import Path
 
 import qmk.path
 
 
 def test_keymap_onekey_pytest():
     path = qmk.path.keymap('handwired/onekey/pytest')
-    assert path == 'keyboards/handwired/onekey/keymaps'
+    assert path.samefile('keyboards/handwired/onekey/keymaps')
 
 
 def test_normpath():
     path = qmk.path.normpath('lib/python')
```

Comment:
`qmk.path.normpath` now expects a Path object:
```suggestion
    path = qmk.path.normpath(Path('lib/python'))
```

Reply:
I've fixed `normpath()` so it now accepts a text string.